docs/Changelog.md

```diff
@@ -9003,6 +9003,38 @@ This version of the operator has been available since version 9 of the default O
 <dd>Constrain index tensor to int64</dd>
 </dl>
 
+### <a name="NonZero-9"></a>**NonZero-9**</a>
+
+  Returns the indices of the elements that are non-zero
```

Comment:
Would you mind to add an example here? The code used in numpy could be changed in the future.